### ISSUE TITLE
 Command "cat"

### ISSUE THREAD

**Issue by DevSmoger** [itzTheMeow/revolt-phlash#2]
https://github.com/itzTheMeow/revolt-phlash/blob/master/src/cmds/cat.ts
This command does not work. Link inside it is old or something.

**Comment by itzTheMeow** [comment]
Yes, `thiscatdoesnotexist.com` just disappeared randomly and I haven't found a decent replacement.

I thought about using  [thesecatsdonotexist.com](http://thesecatsdonotexist.com/) but it would require extra work.

**Comment by DevSmoger** [comment]
Maybe can use some new gallery with random generated images.  
Here is few links I found what should be with free images.
https://image.plus/images/explore https://deepai.org/art https://picogen.io/ https://imggen.ai/ https://aiimag.es/
